src/chainparams.h

```diff
@@ -41,16 +42,16 @@ class CChainParams
         MAX_BASE58_TYPES
     };
 
-    const uint256& HashGenesisBlock() const { return hashGenesisBlock; }
+    const Consensus::Params& GetConsensus() const { return consensus; }
+    const uint256& HashGenesisBlock() const { return consensus.hashGenesisBlock; }
     const MessageStartChars& MessageStart() const { return pchMessageStart; }
     const std::vector<unsigned char>& AlertKey() const { return vAlertPubKey; }
     int GetDefaultPort() const { return nDefaultPort; }
-    const arith_uint256& ProofOfWorkLimit() const { return bnProofOfWorkLimit; }
```

Comment:
I guess we can get rid of these later, by having callers use Params().GetConsensus().GetX(), rather than Params().GetX(). It's fine for now though, assuming it minimizes diffs.